nginx/default.toml

```diff
@@ -19,6 +19,12 @@ worker_processes = 4
 # worker_connections: Connections per Worker Process.  Default = 1024
 worker_connections = 1024
 
+[[include.additional_configuration_file]]
+config_file = "mime.types"
+
+[[include.additional_configuration_file]]
+config_file = "win-utf"
```

Comment:
Agreed - but in the base template, we include mime.types as a "default" include, and now there's mime.types _and_ win-utf being included. Is that cause for alarm?